TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/kafka/KafkaProtobufConsumer.java

```diff
@@ -127,18 +131,21 @@ public List<TrafficStream> readNextTrafficStreamSynchronously() {
                     return null;
                 }
             }).filter(Objects::nonNull);
-            kafkaConsumer.commitSync();
-            return trafficStream.collect(Collectors.toList());
+            return trafficStream.collect(Collectors.<ITrafficStreamWithKey>toList());
         } catch (Exception e) {
             log.error("Terminating Kafka traffic stream");
             throw e;
         }
     }
 
+    @Override
+    public void commitTrafficStream(TrafficStreamKey trafficStreamKey) {
+        kafkaConsumer.commitSync(Map.of());
```

Comment:
Since we are using an empty Map does that mean we are not committing anything here?

Reply:
Good callout - nobody calls that function yet though.  We could throw a NotImplementedException(), but I also had figured that we'd be picking up [this task](https://opensearch.atlassian.net/browse/MIGRATIONS-1379) within a couple days.

Reply:
Gotcha as long as what we plan will cover I'm fine. I just wanted to be clear that currently we aren't doing any manual kafka commits